src/utils/index.js

```diff
@@ -9,7 +25,20 @@ export const setupDocsearch = () => {
     apiKey: "f53a424573adab20d04faa2db150c349",
```

Comment:
Once we have a `.env` file, it would be great to move this public API key over there as well.

Reply:
Totally forgot this comes from [Algolia DocSearch](https://docsearch.algolia.com/) and we don't have a labs equivalent. Please feel free to paste the same key for both prod and labs environments.